### ISSUE TITLE
binary cross entropy loss function

### ISSUE THREAD

**Issue by aimetrics** [hardianlawi/MTL-Homoscedastic-Uncertainty#1]
Hi, @hardianlawi !
I have referenced you code in my click-through rate prediction project. 
However, one of my task could not converge and I have a question about the inconsistency regarding the binary cross entropy loss function and your implementation.
![image](https://user-images.githubusercontent.com/47387000/56331090-71f36300-61bd-11e9-85b2-7008b6f21212.png)
The loss function has added by the log of each task's theta, but you didn't include it in your implementation here:
https://github.com/hardianlawi/MTL-Homoscedastic-Uncertainty/blob/d3aa7e0996f868c6926aa162474d7760a40a0fd7/scripts/estimators.py#L149

**Comment by hardianlawi** [comment]
@aimetrics The equation that you cited is the simplification of the original loss function.

![image](https://user-images.githubusercontent.com/22806872/56332970-f1386500-61c4-11e9-99fb-8c161e96fa54.png)

In the equation you cited, the L2(W) is -log (Softmax (y_2, fW(x)) with fW(x) not scaled. While in my case, I scaled the fW(x) beforehand and directly apply the softmax. They should be equivalent.
